### DISCUSSION TITLE
Commit failed - exit code 128 received, with output: '*** Please tell me who you are

### DISCUSSION THREAD

**Discussion by Spaceball55** [community/community#22224] [New to GitHub]
<p>Hi, I was trying to commit something to my local repository when I had this issue pop up:</p>
<pre><code class="lang-auto">Commit failed - exit code 128 received, with output: '*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: unable to auto-detect email address (got 'User@DESKTOP-O9NT6PU.(none)')
</code></pre>
<p>I had read previous posts here and tried to enter<br>
<code>git config --global user.name "Spaceball55"</code><br>
but this didn’t do anything and the error persisted.</p>
<p>One reason that I am confused is just earlier this week I had no issues committing and pushing to remote repo, but now I can’t even commit to local repo. Any help appreciated!</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The error message say the email address isn’t suitable, so you’ll need to set that too.</p>
<p>If you set that before and the setting is gone now it’s worth investigating <em>why</em> that happened. :wink:</p>

**Comment by Spaceball55** [comment]
<p>Yeah, you were right, I thought I had my email set. I don’t know how it changed, but that is irrelevant. Thank you for your answer! It’s now back to working as it should.</p>
